.github/workflows/korean_grammar.yaml

```
@@ -0,0 +1,12 @@
+name: Check korean grammar on pull request
+
+on: [pull_request]
+
+jobs:
+  build:
```

```suggestion
  grammar_check:
```
해당 라인이 github Action 로그 볼 때 display name입니다. 명시적으로 위와 같이 하면 좋을 듯여